### DISCUSSION TITLE
Cryptic instructions posted with RustDesk server release 1.1.11-1

### DISCUSSION THREAD

**Discussion by ArchPrime** [rustdesk/rustdesk-server#449] [Q&A]
Alongside the  release downloads, there is a very cryptic instruction 

> This version changes default -k of hbbs to -, which means setting key on client side is mandatory. If you want to run empty -k as before, please run hbbs -k "".

I don't want to use encryption, but the above provides absolutely no guidance as to how `hbbs -k ""` should be run.  

On the Windows server version, in an elevated CMD window on the server I stopped the already running hbbs service, then tried typing `hbbs -k ""` - which generated an error :

> An attempt was made to access a socket in a way forbidden by its access permissions. (os error 10013)

I also tried stopping the service again then adding `-k ""` directly to the service start parameters field for the hbbs service in Windows Control Panel/Component Services - but without obvious effect, and when I halted the service again in order to re-check its properties, the start parameters had disappeared.

For the avoidance of doubt and confusion, how specifically do we set up RustDesk Server to work without encryption? (Instructions for a Windows environment would help a lot!)

**Comment by Fox-khb** [comment]
Same problem. Ubuntu 22.04.
I would really like detailed instructions.

**Comment by Dboingue** [comment]
This was a dev memo to devs passing by source code changes.  It is barely more than a diff.
